pkgs/development/tools/tracy/default.nix

```diff
@@ -1,34 +1,38 @@
-{ stdenv, lib, darwin, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig, freetype, Carbon, AppKit }:
+{ stdenv, lib, darwin, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig, freetype, Carbon, AppKit, capstone }:
 
 stdenv.mkDerivation rec {
-  name    = "tracy-${version}";
-  version = "0.6.3";
+  name = "tracy-${version}";
```

Comment:
Maybe switch to `pname`

```suggestion
  pname = "tracy";
```

while at it?

Reply:
:+1: